recipes/brynet/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class BrynetConan(ConanFile):
+    name = "brynet"
+    description = "Header Only Cross platform high performance TCP network library using C++ 11."
+    license = "MIT"
```

Comment:
It also says "996 limited prohibited" I'm the readme

Reply:
I don't understand, I see something related to chinese law about working hours, but it's not a license.

Reply:
![Screenshot_20210605-143856_Chrome](https://user-images.githubusercontent.com/16867443/120902022-02929480-c60c-11eb-967b-c09ce6df88c0.jpg)

https://github.com/996icu/996.ICU/blob/master/LICENSE

Reply:
yes I see, but it's the license of brynet

Reply:
Could have fooled me.

Reply:
It's weird nonetheless. Maybe it's worth to create an issue for them to comment on this?